operator/OWNERS

```diff
@@ -0,0 +1,5 @@
+approvers:
```

Comment:
I can move this to the install directory, however, there will be go code shortly, and will just need to move it back.  Env WG is keen to have an operator for master (which this PR is against).  This PR is targeted at kicking off that effort.

Happy to move to install if you desire given above new information - PLMK.

Cheers
-steve

Reply:
if there is going to be go code, its fine to have a top level directory. That said, since this is specific to kubernetes, don't you think this should be under pkg/kube/operator directory? The secret controller is also present there.

Reply:
@rshriram agreed, kube specific operator code should be under pkg/kube/operator.  WIll make it so.

Cheers
-steve

Reply:
@rshriram would appreciate your expert eye on the parts of the code I've reviewed (also read the commit message please).  I am going to implement this approach for galley.  I think it will be simpler than a full blown process/pod just to instantiate CRDs at runtime.  In the future, this code could serve as a starting point for an operator implementation (in 1.2 timeframe) but I am really keen to make CRDs work well in 1.1 - and time is precious.

Cheers
-steve